examples/websub-internal-hub-sample/subscriber.bal

```diff
@@ -20,11 +20,11 @@ endpoint websub:Listener websubEP {
 }
 service websubSubscriber bind websubEP {
 
-    // Resource accepting intent verification requests.
-    // If this resource is not specified, intent verification would happen automatically - intent would be verified if
-    // the topic specified in the intent verification request matches that specified in the annotation.
+    // Define the resource that accepts the intent verification requests.
+    // If the resource is not specified, intent verification happens automatically. 
+    // It verifies if the topic specified in the intent verification request matches the topic specified as the annotation. 
     onIntentVerification(endpoint caller, websub:IntentVerificationRequest request) {
-        // Build the response for the subscription intent verification request that was received.
+        // Build the response for the subscription-intent-verification request that was received.
```

Comment:
How about "Build the response for the intent verification request that was received for subscription." ?